Author: Jelmer Vernooij <jelmer@openchange.org>
Description: Sort keys when writing mapistore_namedprops.ldif, for consistent output.
Status: Forwarded upstream
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=727008

Index: openchange.zentyal/libmapi/conf/mparse.pl
===================================================================
--- openchange.zentyal.orig/libmapi/conf/mparse.pl	2014-01-29 23:17:48.049271259 +0100
+++ openchange.zentyal/libmapi/conf/mparse.pl	2014-01-29 23:17:48.049271259 +0100
@@ -1182,7 +1182,7 @@
     mparse sprintf "cn: custom";
     mparse sprintf "";
 
-    for my $key ( keys %oleguid ) {
+    for my $key (sort(keys %oleguid)) {
 	my $value = $oleguid{$key};
 
 	mparse sprintf "dn: CN=%s,CN=default", $value;
